File: Python/RemovingStarsFromAString.py
## Leetcode Problem - 2390

def removeStars(s):

    stack = []

    for element in s:
        if element != '*':
            stack.append(element)
        else:
            stack.pop()

    return ''.join(stack)

s = "leet**cod*e"
print(removeStars(s))

File: Javascript/RemovingStarsFromAString.js
// Leetcode Problem - 2390

var removeStars = function (s) {
  const stack = []

  for (const char of s){
    if (char !== '*'){
        stack.push(char)
    }else if (char === '*'){
        stack.pop()
    }
  }

  return stack.join('')
};

const s = "leet**cod*e";
console.log(removeStars(s));
